conans/requirements.txt

```diff
@@ -13,3 +13,4 @@ pygments>=2.0, <3.0
 astroid>=1.6.5
 deprecation>=2.0, <2.1
 tqdm>=4.28.1, <5
+Jinja2==2.10
```

Comment:
Maybe this requirement could be more relaxed? So it minimizes collisions with other possible Jinja2 previously installed.

Reply:
Upps, yes, didn't realize about it after adding it to the requirements.

Quick testing: `sphinx` requires `>=2.3` and our tests are working with it... and Jinja2.3 is from Feb 10, 2010. Maybe go for `Jinja2>=2.3, <3`?